install.sh

```diff
@@ -176,7 +176,7 @@ function install() {
         done
     done
     if [ "$SHOULD_POST_INSTALL" = true ]; then
-      ./post-install.sh --dest-dir "$DEST_DIR" --sysconf-dir "$SYSCONF_DIR" $( (( NO_SUDO == true )) && echo "--no-sudo" )
+      ./post-install.sh --dest-dir "$DEST_DIR" --sysconf-dir "$SYSCONF_DIR" $([ "$NO_SUDO" = true ] && echo "--no-sudo")
```

Comment:
I know I am a bit annoying but could you place the expression in double quotes to avoid "word splitting"? (same for the other one)

Reply:
No problem :+1: